### DISCUSSION TITLE
How to define colors and use them anywhere in the application?

### DISCUSSION THREAD

**Discussion by sinatrocious** [AvaloniaUI/Avalonia#15162] [General]
I want to define a set of colors and reference them.

Adding `Color` to resources (e.g. to resource dictionary):

```xaml
<Color x:Key="color1"/>
```

Will produce following error:

> Value types can only be loaded via converters. We don't want to mess with indirect loads and other weird stuff

How am I supposed to define resources such as colors in Avalonia?

**Comment by workgroupengineering** [comment]
[See](https://github.com/AvaloniaUI/Avalonia/blob/cffb4b8d6230dc8d80e85680899a0fe69b95dd23/src/Avalonia.Themes.Fluent/Accents/BaseColorsPalette.xaml#L23)

**Comment by maxkatz6** [comment]
It's not a color:
```
<Color x:Key="color1"/>
```
This is a color:
```
<Color x:Key="color1">#00ff00</Color>
```

Also, see https://docs.avaloniaui.net/docs/guides/styles-and-resources/resources
